src/_includes/graphql/customer-orders-output.md

```diff
@@ -67,66 +67,25 @@ Attribute | Data type | Description
 
 The deprecated attributes were previously defined in the `CustomerOrder` object in the `customerOrders` query, but have been deprecated for the `customer` query.
 
-#### BundleCreditMemoItem attributes {#BundleCreditMemoItem}
-
-The `BundleCreditMemoItem` object implements the [`CreditMemoItemInterface`](#CreditMemoItemInterface). It also defines the following attribute:
-
-Attribute | Data type | Description
---- | --- | ---
-`bundle_options` | [[ItemSelectedBundleOption]](#ItemSelectedBundleOption) | A list of bundle options that are assigned to the bundle product
-
-#### BundleInvoiceItem attributes {#BundleInvoiceItem}
-
-The `BundleInvoiceItem` object implements the [`InvoiceItemInterface`](#InvoiceItemInterface). It also defines the following attribute:
-
-Attribute | Data type | Description
---- | --- | ---
-`bundle_options` | [[ItemSelectedBundleOption]](#ItemSelectedBundleOption) | A list of bundle options that are assigned to the bundle product
-
-#### BundleOrderItem attributes {#BundleOrderItem}
-
-The `BundleOrderItem` object implements the [`OrderItemInterface`](#OrderItemInterface). It also defines the following attribute:
-
-Attribute | Data type | Description
---- | --- | ---
-`bundle_options` | [[ItemSelectedBundleOption]](#ItemSelectedBundleOption) | A list of bundle options that are assigned to the bundle product
-
-#### BundleShipmentItem {#BundleShipmentItem}
-
-The `BundleShipmentItem` object implements the [`ShipmentItemInterface`](#ShipmentItemInterface). It also defines the following attribute:
-
-Attribute | Data type | Description
---- | --- | ---
-`bundle_options` | [[ItemSelectedBundleOption]](#ItemSelectedBundleOption) | A list of bundle options that are assigned to the bundle product
-
 #### CreditMemo attributes {#CreditMemo}
 
 The `CreditMemo` object contains details about credit memos applied to an order.
 
 Attribute | Data type | Description
 --- | --- | ---
-`comments` | [[SalesCommentItem](#SalesCommentItem)] | Comments on the credit memo
+`comments` | [[CommentItem](#SalesCommentItem)] | Comments on the credit memo
```

Comment:
`[[CommentItem](#SalesCommentItem)]` I don't understand clearly how this is interpreted, but the comments should be of the type SalesCommentItem.

Reply:
SalesCommentItem changes to CommentItem in 2.4.2. I'll create a ticket to change those instances.